sdk/storage/Azure.Storage.Blobs/src/Models/CopyFromUriOperation.cs

```diff
@@ -76,16 +74,20 @@ internal class CopyFromUriOperation : Operation<long>
         /// Initializes a new <see cref="CopyFromUriOperation"/> instance for
         /// mocking.
         /// </summary>
-        public CopyFromUriOperation(
-            string copyId,
-            bool hasCompleted,
-            long? value = default,
-            Response rawResponse = default)
+        protected CopyFromUriOperation()
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new <see cref="CopyFromUriOperation"/> instance
+        /// </summary>
+        /// <param name="client">
+        /// The client used to check for completion.
+        /// </param>
+        /// <param name="id">The ID of this operation.</param>
+        public CopyFromUriOperation(string id, BlobBaseClient client)
```

Comment:
Should we remove the https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/storage/Azure.Storage.Blobs/src/BlobBaseClient.cs#L1541 overloads since we can construct this directly?

Reply:
I'll sync with @KrzysztofCwalina on the final decision here.

Reply:
Yes, removed the methods.